examples/spire/kustomization.yaml

```diff
@@ -16,9 +16,14 @@ configMapGenerator:
   namespace: spire
   files:
   - agent.conf
+- name: k8s-workload-registrar
+  namespace: spire
+  files:
+  - registrar.conf
```

Comment:
Do we plan to use it in overlays? Can we declare this as a configmap?

Reply:
ok, added configMap